docs/schema/validation.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "title": "Validation settings",
+  "markdownDescription": "https://www.mkdocs.org/user-guide/configuration/#validation",
+  "type": "object",
+  "properties": {
+    "nav": {
+      "title": "Settings for validation of navigation",
```

Comment:
Let's just drop the "Settings for", as I think it's redundant. Just "Validation of navigation". Same below.